guiEditor/src/components/commandButtonComponent.tsx

```diff
@@ -18,8 +18,8 @@ export class CommandButtonComponent extends React.Component<ICommandButtonCompon
 
     public render() {
         return (
-            <div className="command-button" onClick={this.props.onClick} title={this.props.tooltip + (this.props.shortcut ? "\n" + this.props.shortcut : "")}>
-                <div className="command-button-icon">
+            <div className={"command-button " + (this.props.isActive ? "active" : "")} onClick={this.props.onClick} title={this.props.tooltip + (this.props.shortcut ? "\n" + this.props.shortcut : "")}>

```

Comment:
Template literals are more efficient when creating a "string out of strings".

```suggestion
            <div className={`command-button ${this.props.isActive ? "active" : ""}} onClick={this.props.onClick} title={`${this.props.tooltip} ${this.props.shortcut ? "\n" + this.props.shortcut : ""`}}>

```

I hope I didn't miss anything. This will generate less strings per reload and is, in general, a good practice.